Content.Shared/Throwing/ThrowingSystem.cs

```diff
@@ -131,16 +136,14 @@ public void TryThrow(EntityUid uid,
         bool recoil = true,
         bool animated = true,
         bool playSound = true,
-        bool doSpin = true)
+        bool doSpin = true,
+        bool unanchor = false)
     {
         if (baseThrowSpeed <= 0 || direction == Vector2Helpers.Infinity || direction == Vector2Helpers.NaN || direction == Vector2.Zero || friction < 0)
             return;
 
-        if ((physics.BodyType & (BodyType.Dynamic | BodyType.KinematicController)) == 0x0)
-        {
-            Log.Warning($"Tried to throw entity {ToPrettyString(uid)} but can't throw {physics.BodyType} bodies!");
+        if ((physics.BodyType & (BodyType.Dynamic | BodyType.KinematicController)) == 0x0 && !unanchor)
```

Comment:
```suggestion
        if (unanchor && HasComp<AnchorableComponent>(uid))
            _transform.Unanchor(uid);

        if ((physics.BodyType & (BodyType.Dynamic | BodyType.KinematicController)) == 0x0)
```

Now that I think about it again, I would move the unanchoring before this body type check. This way it also accounts for cases where unanchoring fails for some reason. And it will be more future proof in case body types are changed in the future.

Remove these two lines below.